examples/backtest.ipynb

```diff
@@ -23,12 +23,14 @@
     "\n",
```

Comment:
<ol><li>To implement this strategy </li><li>list of fold masks.</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
<p><s>with <code>FoldMask</code></s></p>